docs/preview/BYOK/README.md

```diff
@@ -0,0 +1,62 @@
+# Bring Your Own Key (BYOK)
+Bring your own key feature will allow customers to configure customer-managed keys for Azure Container Registry (ACR) encryption. 
```

Comment:
Would also be great to call out the limitation like geo-replication not supported and road map to get them finished

Reply:
Also this will be a public facing doc, please include instructions of how to enable the feature flag for people to use